lib/cc/engine/analyzers/violation.rb

```diff
@@ -26,6 +26,10 @@ def format
           }
         end
 
+        def report_name
+          "#{current_sexp.file}-#{current_sexp.line}"
```

Comment:
I was thinking about that myself, and I'm not sure honestly. I don't think there would be any (or very many at the least) instances where the same start line has multiple end-lines.

I'd love to hear what others think about that though.